var/spack/repos/builtin/packages/cmake/package.py

```diff
@@ -13,7 +13,7 @@ class Cmake(Package):
     url      = 'https://github.com/Kitware/CMake/releases/download/v3.13.0/cmake-3.13.0.tar.gz'
     maintainers = ['chuckatkins']
 
-    version('3.15.1', sha256='cdee72c5ef934c4972f1ad4e9c35712589345f3470a3cf15f7be493c170cc965')
+    version('3.15.1', sha256='18dec548d8f8b04d53c60f9cedcebaa6762f8425339d1e2c889c383d3ccdd7f7')
```

Comment:
@chuckatkins can you confirm that this checksum is correct?

Reply:
This has already been fixed and merged in #12182

Reply:
Thanks for that, I've updated and removed this from the pull request.